stdlib/std.ncl

```diff
@@ -1995,46 +1999,47 @@
         # Examples
 
         ```nickel
-        chars "Hello" =>
-          [ "H", "e", "l", "l", "o" ]
+        std.string.characters "Hello"
+          => [ "H", "e", "l", "l", "o" ]
         ```
       "%
       = fun s => %str_chars% s,
 
     uppercase
       : String -> String
       | doc m%"
-        Returns the uppercase version of the given character (including non-ascii characters) if it exists, the same
-        character if not.
+        Returns the uppercase version of a string. Unicode extended grapheme
+        clusters without an uppercase version are left the same.
```

Comment:
```suggestion
        clusters without an uppercase version are left untouched.
```
"left the same" sounds a bit weird to me. But I don't have a good justification as usual 😅

Reply:
Right, I think I initially used "remains the same" and then wanted to write "left untouched", but ended up with a mix of the two :sweat_smile: